src/docs/style/themes.mdx

```diff
@@ -22,42 +27,97 @@ One of the key benefits of PX Blue is consistency of look & feel between differe
-    caption={'PX Blue offers a light theme (default theme) and a dark theme (for low-light condition usage).'}
-/>
+<div style={REGULAR_WIDTH_STYLE}>
+    <DemoCard
+        repository={'showcase-demo'}
+        angular={`https://stackblitz.com/github/pxblue/angular-showcase-demo/tree/master`}
+        react={`https://codesandbox.io/s/github/pxblue/react-showcase-demo/tree/master`}
+    />
+    <Divider />
+</div>
 
-The PX Blue themes packages includes two distinct themes:
+# Blue Theme / Light Theme
 
--   **Blue** is the default theme for all PX Blue applications. It should be applied at the root level of your app. It provides a variety of shades of blue for use in buttons, toolbars, etc. The standard background color is a light gray.
--   **Blue Dark** is complementary to the Blue theme. It is intended primarily for applications used in low-light conditions. The primary colors used in the Blue Dark theme are various shades of gray with blue accents. It can be used as a standalone theme or combined with the Blue (default) theme to provide contrast for areas such as detail sidebars.
+<div style={REGULAR_WIDTH_STYLE}>
+    <ImageGrid images={[LightThemeExample]} regularWidth caption={'An example of a light-themed application.'} />
+</div>
 
-<DemoCard
-    repository={'showcase-demo'}
-    angular={`https://stackblitz.com/github/pxblue/angular-showcase-demo/tree/master`}
-    react={`https://codesandbox.io/s/github/pxblue/react-showcase-demo/tree/master`}
-/>
+The blue theme is the default theme for all PX Blue applications. With a clean color vibrance and high text legibility, it is designed to suit day-to-day use on a computer monitor, on-site facility inspection with a tablet, and on a mobile phone screen. It should be applied at the root level of your app.
 
-> **NOTE:** Combining multiple themes in your application is optional. You may stick to using a single theme (Blue) for your entire application and then apply one-off styling where necessary to achieve other effects.
+The light theme uses our iconic [Blue 500](/style/color) color (`#007bc1`) throughout the application, notably in toolbars, buttons, and icons. In some places, Light Blue 500 is applied as an accent/secondary color on interactive UI elements such as form controls. The light theme's base background color is White 200, although the surface elements are often shaded in White 50 (pure white).
 
+<ImageGrid images={[LightThemeColors]} regularWidth />
+<Divider />
+
+# Dark Theme / Blue Dark Theme
+
+<div style={REGULAR_WIDTH_STYLE}>
+    <ImageGrid images={[DarkThemeExample]} regularWidth caption={'An example of a dark-themed application.'} />
+</div>
+
+The dark theme is intended primarily for applications used in low-light conditions. As the name indicates, it utilizes different shades of dark colors (grays and blacks) and uses other colors only for interactive components and status displays.
+
+<ImageGrid images={[DarkThemeColors]} regularWidth />
```

Comment:
I can do that. I feel like since it is explained by the paragraph immediately following the image, I kinda skipped it here.

Reply:
Yeah, I know what you mean. It just catches the reader (me) by surprise to see the color on the left and then a different color on the right. Maybe we could integrate some error text in the example on the right in both light/dark and refer to that as the error color instead of the badge (which isn't actually necessarily related to warnings if you think about it).